Report.md

```diff
@@ -8,6 +8,10 @@ Link to Project GitHub repository: [Tutorial10_Group-1](https://github.com/zacaz
 
 Problem Definition
 ==================
+Customer experience is heavily impacted because of the call flow rate  in the (CMC) as customers are not being assigned to well informed RMs that have the  most appropriate knowledge about that particular holiday destination and its traditions i.e. their skillset, as well ability to cater to that Customer’s profile. This in turn is impacting RMs performance in selling packages and serving customers effectively and efficiently.  
+There is also not a defined guideline for RMs to approach outbound calls which effects the service provided to the end customer. High costs associated with inbound calls from Customers due to the per-call handling time is also something that needs to be factored in. 
+Furthermore, a key issue faced by customers is waiting times during busy times this can have a negative impact on the major  travel company’s brand and make them potentially loose customers. 
```

Comment:
```suggestion
Furthermore, a key issue faced by customers is waiting times during busy times. This can have a negative impact on the major travel company’s brand and make them potentially lose customers. 
```

Reply:
```suggestion
Furthermore, a key issue faced by customers is waiting times during busy times this can have a negative impact on the major travel company’s brand and make them potentially lose customers. 
```